src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'books/ADD_BOOK';
+const REMOVE_BOOK = 'books/REMOVE_BOOK';
```

Comment:
- [ ] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:    const ADD_BOOK = bookstore-ivanmvh/books/ADD_BOOK and const REMOVE_BOOK = bookstore-ivanmvh/books/REMOVE_BOOK